client/landing/gutenboarding/components/locale-context/index.tsx

```diff
@@ -48,12 +49,14 @@ export const ChangeLocaleContextConsumer = ChangeLocaleContext.Consumer;
 
 export const LocaleContext: React.FunctionComponent = ( { children } ) => {
 	const [ contextLocaleData, setContextLocaleData ] = React.useState< LocaleData | undefined >();
+	const [ localeDataLoaded, setLocaleDataLoaded ] = React.useState< boolean >( false );
```

Comment:
Just a TypeScript thing: you usually only provide explicit type arguments (like `< boolean >`) when they're necessary, and in this case it can be omitted now and `localeDataLoaded` will still be a `boolean`.

```suggestion
	const [ localeDataLoaded, setLocaleDataLoaded ] = React.useState( false );
```